### PR TITLE
Repository Interface Cleanup

### PR DESCRIPTION
**Pull request by vjkoskela**
Rename repository interface methods to be more consistent with each other and avoid ambiguity/collisions (e.g. get on report and job).

### CODE REVIEW

File: app/com/arpnetworking/kairos/client/KairosDbClient.java [hunk 1]
```diff
@@ -31,9 +31,9 @@
  */
 public interface KairosDbClient {
     /**
-     * Executes a query for datapoints from  KairosDB.
+     * Executes a queryAlerts for datapoints from  KairosDB.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/KairosDbClient.java]: I don't think changes in this file were intended.

File: app/com/arpnetworking/kairos/client/KairosDbClientImpl.java [hunk 2]
```diff
@@ -178,9 +178,9 @@ private KairosDbClientImpl(final Builder builder) {
     private final URI _uri;
     private final FiniteDuration _readTimeout;
 
-    static final URI METRICS_QUERY_PATH = URI.create("/api/v1/datapoints/query");
+    static final URI METRICS_QUERY_PATH = URI.create("/api/v1/datapoints/queryAlerts");
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/KairosDbClientImpl.java]: I don't think changes in this file were intended.

File: app/com/arpnetworking/kairos/client/models/Aggregator.java [hunk 3]
```diff
@@ -28,7 +28,7 @@
 import javax.annotation.Nullable;
 
 /**
- * Model class to represent the aggregator in a metrics query.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/Aggregator.java]: I don't think this change was intended.

File: app/com/arpnetworking/kairos/client/models/Metric.java [hunk 4]
```diff
@@ -29,7 +29,7 @@
 import javax.annotation.Nullable;
 
 /**
- * Holds the data for a Metric element of the query.
+ * Holds the data for a Metric element of the queryAlerts.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/Metric.java]: I don't think this change was intended.

File: app/com/arpnetworking/kairos/client/models/MetricsQuery.java [hunk 5]
```diff
@@ -34,13 +34,13 @@
 import javax.annotation.Nullable;
 
 /**
- * Model class to represent a metrics query.
+ * Model class to represent a metrics queryAlerts.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/MetricsQuery.java]: I don't think this change was intended.

File: app/com/arpnetworking/kairos/client/models/MetricsQueryResponse.java [hunk 6]
```diff
@@ -33,7 +33,7 @@
 import java.util.List;
 
 /**
- * Model class to represent a metrics query response from KairosDB.
+ * Model class to represent a metrics queryAlerts response from KairosDB.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/MetricsQueryResponse.java]: I don't think this change was intended.

File: app/com/arpnetworking/kairos/client/models/Rollup.java [hunk 7]
```diff
@@ -21,7 +21,7 @@
 import net.sf.oval.constraint.NotNull;
 
 /**
- * Defines a query and destination for a RollupTask.
+ * Defines a queryAlerts and destination for a RollupTask.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/Rollup.java]: I don't think this change was intended.

File: app/com/arpnetworking/kairos/client/models/RollupQuery.java [hunk 8]
```diff
@@ -57,7 +57,7 @@ public Builder() {
         }
 
         /**
-         * Sets the relative start time of the query. Required. Cannot be null.
+         * Sets the relative start time of the queryAlerts. Required. Cannot be null.
```
Comment (BrandonArp) [inline app/com/arpnetworking/kairos/client/models/RollupQuery.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/AlertRepository.java [hunk 9]
```diff
@@ -56,23 +56,23 @@
      * @param organization The organization owning the alert.
      * @return The number of alerts deleted.
      */
-    int delete(UUID identifier, Organization organization);
+    int deleteAlert(UUID identifier, Organization organization);
 
     /**
-     * Create a query against the alerts repository.
+     * Create a queryAlerts against the alerts repository.
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/AlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java [hunk 10]
```diff
@@ -55,7 +55,7 @@
     /**
      * Public constructor.
      *
-     * @param cassandraSession a Session to use to query data
+     * @param cassandraSession a Session to use to queryAlerts data
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java [hunk 11]
```diff
@@ -116,17 +116,17 @@ public int delete(final UUID identifier, final Organization organization) {
     }
 
     @Override
-    public AlertQuery createQuery(final Organization organization) {
+    public AlertQuery createAlertQuery(final Organization organization) {
         assertIsOpen();
         LOGGER.debug()
-                .setMessage("Preparing query")
+                .setMessage("Preparing queryAlerts")
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 12]
```diff
@@ -124,24 +124,24 @@ public void close() {
     }
 
     @Override
-    public AlertQuery createQuery(final Organization organization) {
+    public AlertQuery createAlertQuery(final Organization organization) {
         assertIsOpen();
         LOGGER.debug()
-                .setMessage("Preparing query")
+                .setMessage("Preparing queryAlerts")
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 13]
```diff
@@ -124,24 +124,24 @@ public void close() {
...
     }
 
     @Override
-    public QueryResult<Alert> query(final AlertQuery query) {
+    public QueryResult<Alert> queryAlerts(final AlertQuery query) {
         assertIsOpen();
         LOGGER.debug()
                 .setMessage("Querying")
-                .addData("query", query)
+                .addData("queryAlerts", query)
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 14]
```diff
@@ -124,24 +124,24 @@ public void close() {
...
                 .log();
 
-        // Create the base query
+        // Create the base queryAlerts
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 15]
```diff
@@ -296,7 +296,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
     private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseAlertRepository.class);
 
     /**
-     * Inteface for database query generation.
+     * Inteface for database queryAlerts generation.
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 16]
```diff
@@ -305,7 +305,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
          *
          * @param repository The alert repository instance.
          * @param query The repository agnostic <code>AlertQuery</code>.
-         * @return The database specific <code>PagedList</code> query result.
+         * @return The database specific <code>PagedList</code> queryAlerts result.
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 17]
```diff
@@ -328,7 +328,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
     }
 
     /**
-     * RDBMS agnostic query for alerts.
+     * RDBMS agnostic queryAlerts for alerts.
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: I don't think this change was intended.

**Comment by vjkoskela** [comment]
Apologies for that, looks like my IDE got a little carried away. I've replaced it with a version where I just replaced the method calls (setting on the refactoring UI) and checked the incremental diff of each method rename to ensure there was no funny business.

**Pull request merged**